tests/IResearch/IResearchAnalyzerFeature-test.cpp

```diff
@@ -532,10 +532,10 @@ SECTION("test_persistence") {
 
   // ensure there is an empty configuration collection
   {
-    auto* collection = vocbase->lookupCollection("_iresearch_analyzers");
+    auto collection = vocbase->lookupCollection("_iresearch_analyzers");
 
     if (collection) {
-      vocbase->dropCollection(collection, true, -1);
+      vocbase->dropCollection(collection.get(), true, -1);
     }
```

Comment:
shouldn't dropCollection(...) also allow shared_ptr ?

Reply:
I tried to keep the changes to a minimum otherwise there are too many files in the changeset
although I do agree that eventually raw pointers should be shared pointers